pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -1658,6 +1660,37 @@ func TestDurationToHAProxyTimespec(t *testing.T) {
 	}
 }
 
+func TestCapReloadIntervalValue(t *testing.T) {
+	testCases := []struct {
+		inputDuration  time.Duration
+		expectedOutput time.Duration
+	}{
+		// Values below the minimum 1s returns 1s.
+		{1 * time.Millisecond, 1 * time.Second},
+		{-1, 1 * time.Second},
+
+		// Values above the maximum 120s returns 120s.
+		{6 * time.Minute, 120 * time.Second},
+		{1 * time.Hour, 120 * time.Second},
+		{365 * time.Second, 120 * time.Second},
+
+		// Values in the allowed range returns itself (i.e. b/w 1s and 5 mins).
```

Comment:
If you need to change anything else, you should update this comment, which has the wrong range.